testsuite/tests/kuadrant/authorino/operator/sharding/conftest.py

```diff
@@ -0,0 +1,41 @@
+"""Conftest for authorino sharding tests"""
+import pytest
+
+from testsuite.openshift.envoy import Envoy
+from testsuite.openshift.objects.auth_config import AuthConfig
+
+
+@pytest.fixture(scope="module")
+def envoy(request, authorino, openshift, blame, backend, ):
+    """Envoy"""
+
+    def _envoy(auth=authorino):
+        envoy = Envoy(openshift, auth, blame("envoy"), blame("label"), backend.url)
+        request.addfinalizer(envoy.delete)
+        envoy.commit()
+        return envoy
+
+    return _envoy
+
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def authorization(request, authorino, blame, openshift, module_label):
+    """In case of Authorino, AuthConfig used for authorization"""
+
+    def _authorization(hostname=None, sharding=None, response=None):
```

Comment:
I would rename `sharding` to something more verbose, like `label`. Also I think we can remove `response` and use the value from `label` instead.